tests/update/updates_packagekit_gpk.pm

```diff
@@ -32,6 +32,14 @@ sub turn_off_screensaver {
     send_key("ctrl-d");
 }
 
+sub tell_packagekit_to_quit {
+    # tell the PackageKit daemon to stop in order to next load with new libzypp
+    # this is different from pkcon_quit
+    x11_start_program('xterm');
+    script_run("pkcon quit");
```

Comment:
Next time pleas use either `assert_script_run` by default or explicitly check the return value